$NetBSD: patch-aa,v 1.1 1998/04/10 22:30:41 tron Exp $

Turn LLNL's compile script into a configure script :-)

		-- tsarna@endicor.com, 18 March 1998

--- ../Tools/compile.py.orig	Tue Mar 17 23:28:14 1998
+++ ../Tools/compile.py	Tue Mar 17 23:28:14 1998
@@ -632,7 +632,7 @@
 
     compileall.compile_dir(".")
     os.system("make -f Makefile.pre.in boot")
-    os.system("make")
+#    os.system("make")
 
 if sys.platform == 'win32':
 	swig_found = find_swig()
